homeassistant/components/ecowitt/const.py

```diff
@@ -0,0 +1,50 @@
+"""Constants used by ecowitt component."""
+
+from homeassistant.const import CONF_UNIT_SYSTEM_IMPERIAL, CONF_UNIT_SYSTEM_METRIC
+
+ECOWITT_PLATFORMS = ["sensor"]
```

Comment:
No need to set the `ECOWITT_` prefix:
```suggestion
PLATFORMS = ["sensor"]
```